metadata_service/__init__.py

```diff
@@ -86,7 +86,15 @@ def create_app(*, config_module_class: str) -> Flask:
                      '/tags/')
     api.add_resource(UserDetailAPI,
                      '/user/<path:user_id>')
-
+    api.add_resource(UserBookmarkAPI,
+                     '/user/<path:user_id>/bookmark/',
+                     '/user/<path:user_id>/bookmark/<path:table_uri>')
```

Comment:
For starter, how about adding type here?
`/user/<path:user_id>/bookmark/<type>/<path:table_uri>`
I think we can simplify the implementation by checking type is always table for now.

Reply:
I change it, but call it resource_type as type is a key_word in python.